src/Microsoft.ML.Transforms/Text/doc.xml

```diff
@@ -179,7 +179,58 @@
     <example name="LightLDA">
       <example>
         <code language="csharp">
-          pipeline.Add(new LightLda(("InTextCol" , "OutTextCol")));
+          pipeline.Add(new LightLda((&quot;InTextCol&quot; , &quot;OutTextCol&quot;)));
+        </code>
+      </example>
+    </example>
+
+    <member name="WordEmbeddings">
+      <summary>
+        Word Embeddings transform is a text featurizer which converts vectors of text tokens into sentence vectors using a pre-trained model.
+      </summary>
+      <remarks>
+        WordEmbeddings wrap different embedding models, such as GloVe. Users can specify which embedding to use. 
+        The available options are various versions of <a href="https://nlp.stanford.edu/projects/glove/">GloVe Models</a>, <a href="https://en.wikipedia.org/wiki/FastText">FastText</a>, and <a href="http://anthology.aclweb.org/P/P14/P14-1146.pdf">Sswe</a>.
+        <para>
+          Note: As WordEmbedding requires a column with text vector, e.g. %3C%27This%27, %27is%27, %27good%27%3E, users need to create an input column by:
+          <list type="bullet">
+          <item><description>concatenating columns with TX type,</description></item>
```

Comment:
Concatenating columns of single unigrams is very unlikely.

We should be recommending _only_ to use `output_tokens=True` in `NGramFeaturizer()`. All of our current pre-trained models require tokens which are lowercased unigrams w/ diacritics removed (which are the defaults for NGramFeaturizer).

Reply:
Thank you for clarification, didn't knew about this.
Will update documentation in later PR.

---
In reply to: [204193596](https://github.com/dotnet/machinelearning/pull/545#discussion_r204193596) [](ancestors = 204193596)